src/applications/bmqstoragetool/m_bmqstoragetool_journalfileprocessor.t.cpp

```diff
@@ -2402,6 +2416,29 @@ static void test24_summaryWithQueueDetailsTest()
 
     // Run search
     searchProcessor->process();
+
+    // Prepare expected output
+    bmqu::MemOutStream expectedStream(bmqtst::TestHelperUtil::allocator());
+    expectedStream << "5 message(s) found.\n";
+    bsl::vector<bsl::string> fields(bmqtst::TestHelperUtil::allocator());
+    fields.push_back("Number of partially confirmed messages");
+    fields.push_back("Number of confirmed messages");
+    fields.push_back("Number of outstanding messages");
+    bmqu::AlignedPrinter printer(expectedStream, fields);
+    printer << 3 << 2 << 2;
+    expectedStream << "Outstanding ratio: 40% (2/5)\n";
+
+    expectedStream << "Total number of records: 15\n"
+                      "Number of records per Queue:\n"
+                      "    Queue Key             : 6162636465\n"
+                      "    Total Records         : 15\n"
+                      "    Num Queue Op Records  : 0\n"
+                      "    Num Message Records   : 5\n"
+                      "    Num Confirm Records   : 5\n"
+                      "    Num Delete Records    : 5";
+
+    bsl::string res(resultStream.str(), bmqtst::TestHelperUtil::allocator());
+    ASSERT(res.starts_with(expectedStream.str()));
```

Comment:
A general comment, in the PR's description there are much more additions than deletions: `+662 −72`
It looks suspicious because you can expect the numbers to be pretty close if you are doing code refactor. If you look closely at the changed files, many of them have additions only. This means that the PR is adding new code rather than changing the existing one. I think the reason for this is the conflict resolution process when the code moved to another places in the main branch. You should probably search for the place where the code moved and apply changes there.
If you see places where the code was just added without modification of other code, it's probably a place where the conflict resolution went wrong